travis/before_install.s390x-gcc.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+set -ev
```

Comment:
Could you please dump auxiliary vector via  `LD_SHOW_AUXV=1 /bin/true` to the ZARCH version?

Reply:
Done.